forgerock-authenticator/ios/Classes/FRAClientWrapper.swift

```diff
@@ -342,7 +354,11 @@ open class FRAClientWrapper {
             }
         } onError: { (error) in
             if let result {
-                result(FlutterError(code: "HANDLE_NOTIFICATION_EXCEPTION", message: error.localizedDescription, details: nil))
+                if case AccountError.accountLocked(let param) = error {
+                    result(FlutterError(code: "ACCOUNT_LOCK_EXCEPTION", message: "This account is locked. It violates the following policy: (\(param))", details: nil))
```

Comment:
"This account is locked. It violates the following policy: (\(param))" is hardcoded in 3 places. Do we have a file for strings? Even if no, we still can have a variable here maybe so that we don't repeat it in 3 places? Easier to change later.

Reply:
good suggestion, we create a private function to handle this